Resources/Prototypes/Loadouts/Jobs/Civilian/musician.yml

```
@@ -6,4 +6,257 @@
 - type: startingGear
   id: MusicianWintercoat
   equipment:
-    outerClothing: ClothingOuterWinterMusician
\ No newline at end of file
+    outerClothing: ClothingOuterWinterMusician
+
+# Instruments
+- type: loadout
+  id: Glockenspiel
+  equipment: Glockenspiel
+
+- type: startingGear
+  id: Glockenspiel
+  equipment:
+    outerClothing: GlockenspielInstrument
```

you should change all startingGear like this
```suggestion
- type: startingGear
  id: Glockenspiel
  storage:
    back: 
    - GlockenspielInstrument
```
so it will spawn in your backpack, otherwise it cannot spawn on your outerclothing slot and just would be dropped on floor

---

alright, sick, fixed